Mlem/App/Views/Shared/ExpandedPostView.swift

```diff
@@ -54,16 +51,55 @@ struct ExpandedPostView: View {
     
     @ViewBuilder
     func content(for post: any Post1Providing) -> some View {
-        FancyScrollView {
-            LazyVStack(alignment: .leading, spacing: 0) {
-                LargePostView(post: post, isExpanded: true)
-                Divider()
-                ForEach(comments.reduce([]) { $0 + $1.tree() }) { comment in
-                    CommentView(comment: comment)
-                        .transition(.move(edge: .top).combined(with: .opacity))
-                        .zIndex(1000 - Double(comment.depth))
+        ScrollViewReader { proxy in
+            FancyScrollView {
+                LazyVStack(alignment: .leading, spacing: 0) {
+                    LargePostView(post: post, isExpanded: true)
+                    Divider()
+                    ForEach(comments.reduce([]) { $0 + $1.tree() }) { comment in
+                        CommentView(comment: comment, highlight: showCommentWithId == comment.id)
+                            .transition(.move(edge: .top).combined(with: .opacity))
+                            .zIndex(1000 - Double(comment.depth))
+                    }
+                }
+                .animation(.easeInOut(duration: 1.5), value: showCommentWithId)
+            }
+            .onAppear {
+                if let showCommentWithId {
+                    // The scroll destination isn' always accurate. Possibly due to the post image changing
+                    // size on load? Using `anchor: .top` would be better here, but `anchor: .center` makes
+                    // the inaccuracy less noticeable. See also the comment further up the file.
+                    proxy.scrollTo(showCommentWithId, anchor: .center)
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+                        self.showCommentWithId = nil
+                    }
```

Comment:
This solution seems good for now--when we have the image size from `PostView`, we should be able to set the image size correctly before it loads and thereby fix this properly